### PR TITLE
Og to group

### PR DESCRIPTION
**Pull request by emerham**
Migrations of User accounts filtered to a set of roles,
Process plugin to manually map old roles to new roles.
Migrations of OG to Group.
Migrations of OG Memberships to Group memberships.

### CODE REVIEW

**Review by mechanoid23 — APPROVED** [review]
Okay. I think I understand what you're doing here.
With the user filtering based on roles, is the intention to only migrate users that have roles, leaving basic 'authenticated' users behind?

**Comment by emerham** [comment]
@mechanoid23 Yes, we don't want every account in the site to be migrated as most of those accounts only exist for the user profile and they don't do any content editing.

File: modules/og_to_group/og_to_group.module [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+use Drupal\migrate\Row;
+
+/**
+ * Implements hook_migrate_prepare_row().
+ *
+ * Get the Book page that is the old Group landing page and take it's alias and
+ * apply it to the new group alias.
+ */
+function og_to_group_migrate_prepare_row(Row $row, MigrateSourceInterface $source, MigrationInterface $migration) {
+  switch ($migration->id()) {
```
Comment (Ruefa) [inline modules/og_to_group/og_to_group.module:15]: why use a switch with one case?

**Review by Ruefa — APPROVED** [review]
Looks good left 1 comment

File: modules/og_to_group/og_to_group.module (hunk 1 above)
Comment (emerham) [reply modules/og_to_group/og_to_group.module:15]: Copied from drupal.org documentation. I believe they showed using a switch as you might want to use the prepare row on multiple migrations. Performance should be the same using an if verses a switch.

**Pull request merged**